src/components/Filters/SearchGridFilter.vue

```diff
@@ -132,6 +132,10 @@ export default {
     color: #333333;
   }
   &__visible {
+    @include desktop {
```

Comment:
You're missing `@import "node_modules/bulma/sass/utilities/_all";` in the beginning of the styles